_includes/manuals/nightly/3.1.3_puppet_versions.md

```diff
@@ -1,84 +1,45 @@
-Foreman integrates with Puppet and Facter in a few places, but generally using a recent, stable version will be fine.  The exact versions of Puppet, Puppet Server and Facter that Foreman is compatible with are listed below.
+Foreman integrates with Puppet and Facter in a few places, but generally using a recent, stable version will be fine.  The exact versions of Puppet, Puppetserver and Facter that Foreman is compatible with are listed below.
 
 <table class="table table-bordered table-condensed">
   <tr>
     <th>Puppet version</th>
-    <th>Foreman installer</th>
-    <th>Smart proxy</th>
-    <th><a href="/manuals/{{page.version}}/index.html#3.5.4PuppetReports">Report/fact processors</a></th>
-    <th><a href="/manuals/{{page.version}}/index.html#3.5.5FactsandtheENC">External node classifier</a></th>
+    <th>Foreman installer (AIO)</th>
+    <th>Foreman installer (non-AIO)</th>
+    <th>Smart Proxy</th>
   </tr>
   <tr>
-    <td>0.x-3.x</td>
+    <td>0.x-4.3</td>
+    <td>Not supported</td>
+    <td>Not supported</td>
     <td>Not supported</td>
-    <td>Untested</td>
-    <td>Untested</td>
-    <td>Untested <span class='footnote'>*</span></td>
   </tr>
   <tr>
-    <td>4.x (AIO)</td>
+    <td>4.4-4.10</td>
```

Comment:
Technically the Smart Proxy should work with 4.4+ since the API hasn't changed. Do we want to advertise that or just say it's unsupported?